css/main.css

```diff
@@ -167,28 +172,29 @@ span {
 
 /* * Margin styles here */
 
-.mar-8p{
+.marg-8p {
```

Comment:
You can form separate file utiity.css for general margin and style

Reply:
Created a separate file for utility classes.